web/cypress/e2e/pages/studies.cy.ts

```diff
@@ -0,0 +1,70 @@
+beforeEach(() => {
+  cy.clearCookies();
+  cy.clearLocalStorage();
+});
+
+describe(`Studies page content`, () => {
+  it("should show base role content", () => {
+    cy.loginAsBase();
+    cy.mockAuthAsBaseUser();
+    cy.visit("/studies");
+    cy.waitForAuth();
+
+    cy.contains("This page is being built. Please check back soon for updates!").should("be.visible");
+  });
+
+  it("should show content for base approved researcher", () => {
+    cy.loginAsBase();
+    cy.mockAuthAsBaseApprovedResearcher();
+    cy.visit("/studies");
+    cy.waitForAuth();
+
+    cy.contains("This page is being built. Please check back soon for updates!").should("not.exist");
+    cy.contains("Your Studies").should("be.visible");
+  });
+});
+
+describe("Approved researcher can create studies", () => {
+  beforeEach(() => {
+    cy.loginAsBase();
+    cy.mockAuthAsBaseApprovedResearcher();
+    cy.visit("/studies");
+    cy.waitForAuth();
+  });
+
+  it("can bring up the form", () => {
+    cy.contains("Create Study").should("be.visible").click();
+    cy.get("[data-cy='create-study-form']").should("be.visible");
+  });
+
+  it("owner set to user & can't be edited", () => {
+    cy.fixture("auth-base-approved-researcher.json").then((data) => {
```

Comment:
this doesn't exist(?). if just getting the base user it can be imported? https://github.com/ucl-arc-tre/portal/blob/b1c3c7fb1cc8f8425a7b1cc99f7f439142c40ffa/web/cypress/e2e/integration/people/admin-editing.cy.ts#L1

Reply:
tried that initially but the mocking uses that [fixture](https://github.com/ucl-arc-tre/portal/blob/b1c3c7fb1cc8f8425a7b1cc99f7f439142c40ffa/web/cypress/fixtures/auth-base-approved-researcher.json)

Reply:
oh fun (sorry – my grep was not good!)